packages/plugins/coverage/src/lib/index.ts

```diff
@@ -25,7 +25,7 @@ export default class Coverage {
       return;
     }
 
-    this.embark.registerActionForEvent("tests:contracts:compile:before", async (contractsFiles, cb) => {
+    this.embark.registerActionForEvent("tests:contracts:compile:before", undefined, async (contractsFiles, cb) => {
```

Comment:
I might be missing something here, but this seems to be a change we don't want? In fact, it looks like this will turn off the actual action here.

Reply:
It's required for TS to be happy since we previously updated the signature to accept a `{ priority: integer }` option

Reply:
Okay, as discussed offline, this parameter was introduced due to the new `priority` API introduced in https://github.com/embark-framework/embark/commit/776db1b7f71e9a78f216cf2acc6c1387c60b3604

However, there's a bug in our typings that forces users to do this.

Sent a fix here: https://github.com/embark-framework/embark/pull/2060

Once that lands, the `undefined` can be removed here

/cc @emizzle